frontend/public/locales/en/translation.json

```diff
@@ -1194,6 +1194,9 @@
   "discoveryConfig.add": "Create",
   "discoveryConfig.autoImportPolicy.Info": "After discovery settings are configured, discovered Red Hat OpenShift on AWS clusters can be automatically imported as managed clusters using policy enforcement. Policies used in this way can also be configured with filters or annotations to specify which clusters should be automatically imported.",
   "discoveryConfig.cancel": "Cancel",
+  "discoveryConfig.ClusterTypes.label": "Discovery configuration cluster types",
+  "discoveryConfig.ClusterTypes.labelHelp": "discoveryConfig.discoveryClusterTypes.labelHelp",
+  "discoveryConfig.ClusterTypes.placeholder": "Select a Discovery configuration cluster type",
```

Comment:
Select "a"? Isn't this a multiselect field? (also, same comment here about redundancy... no need to include "Discovery configuration" here)